docs/reference/transform/apis/put-transform.asciidoc

```diff
@@ -128,6 +128,23 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=pivot-group-by]
 ====
 //End pivot
 
+//Begin settings
```

Comment:
:+1:  sorry for the bad placement! I see, it should be ordered, will do better next time!

Reply:
No worries!